lib/activity.js

```diff
@@ -25,9 +25,10 @@ export class ActivityClient extends Client {
             .then(data => this.afterDataProcessed(endpoint, data));
     }
 
-    getLiveFeed(opts = {}) {
-        const endpoint = this.getEndpoint('getLiveFeed', opts);
-        return this.fetch(endpoint);
+    getActivities(opts = {}) {
+        const endpoint = this.getEndpoint('getActivities', opts);
+        return this.fetch(endpoint)
```

Comment:
you sure you don't need to run .then(data => data.res()) is .res just undefined? might be worth asking jono

Reply:
checked with jono all good returns an array